src/Data/List/Relation/Binary/Suffix/Homogeneous/Properties.agda

```diff
@@ -45,3 +45,20 @@ isDecPartialOrder dpo = record
   ; _≟_            = Pointwise.decidable DPO._≟_
   ; _≤?_           = suffix? DPO._≤?_
   } where module DPO = IsDecPartialOrder dpo
+
+
+module _ {A : Set a} where
+
+  open import Data.List.Base using (List; []; _∷_)
+  open import Data.List.Properties using (++-monoid)
+  open import Algebra.Properties.Monoid.Divisibility (++-monoid A)
+  open import Relation.Binary.PropositionalEquality using (_≡_; refl; cong)
+  open import Relation.Binary using (_⇒_)
+
+  Suffix-as-∣ʳ : Suffix _≡_ ⇒ _∣ʳ_
```

Comment:
Should these lemmas live in a new file `Suffix.Propositional` and `Suffix.Propositional.Properties`?

Reply:
Fair; moved everything to `Suffix.Propositional.Properties`.
I did not define `Suffix.Propositional` because, just like the non-existent `Suffix.Homogeneous`,
it would be a single definition of an alias for the type constructor, wouldn't it?

Reply:
Yup that seems fair!